include/picongpu/particles/creation/creation.kernel

```diff
@@ -382,58 +399,24 @@ namespace creation
                      * - then the shared counter is set back by one frame size
                      * - sync so that every thread knows about the new frame
                      */
-                    onlyMaster(
+                    onlyMasters(
                         [&](
-                            uint32_t const,
+                            uint32_t const linearIdx,
                             uint32_t const
                         )
                         {
-                            if( newFrameFillLvl >= maxParticlesInFrame )
+                            if( linearIdx == 0  && newFrameFillLvl >= maxParticlesInFrame )
                             {
-                                targetFrame = targetBox.getEmptyFrame( );
-                                targetBox.setAsLastFrame(
-                                    acc,
-                                    targetFrame,
-                                    block
-                                );
-                                newFrameFillLvl -= maxParticlesInFrame;
+                                 newFrameFillLvl -= maxParticlesInFrame;
+                                 // copy the not filled frame pointer to the beginning
+                                 targetFrames[ 0 ] = targetFrames[ 1 ];
+                                 // reset second frame
+                                 targetFrames[ 1 ] = nullptr;
```

Comment:
By removing the second creation step the documentation `< CREATE 1 >` can be just changed to `< CREATE >` and `< CREATE 2 >` [here](https://github.com/ComputationalRadiationPhysics/picongpu/pull/2258/files#diff-adb193a5d6509402921e4732bc77dd78R360) can be removed

Reply:
changes to `< create>`